tox.ini

```diff
@@ -25,6 +25,13 @@ commands =
     pyflakes ./docs/conf.py
 deps = flake8
 
+[testenv:format]
```

Comment:
I suggest including this in the `flakes` job (and possibly renaming it `lint`) as the test matrix is already a pretty hefty 36 jobs.

And fixing Black often fixes Flake8 too.

Reply:
Great idea!